variables.tf

```
@@ -316,3 +316,22 @@ variable "storage_accounts" {
     error_message = "Storage account type must be either \"AzureFiles\" or \"AzureBlob\"."
   }
 }
+
+variable "public_network_access_enabled" {
+    description = "Should public access be enabled for this Web App?"
+    default = true
+    type = bool
+}
+
+
+variable "client_certificate_mode" {
+  description = "Should client cerftificate mode be enabled for this Web App?"
+  default = "Optional"
+  type = string
+}
```

- Update order of arguments
- Update description
- Update default value (feel free to argue against this change)
- Update formatting

```suggestion
variable "client_certificate_mode" {
  description = "The client cerftificate mode for this Web App. Value must be \"Required\", \"Optional\" or \"OptionalInteractiveUser\"."
  type        = string
  default     = "Required"
}
```